tests/unit/test_config.py

```diff
@@ -402,3 +402,11 @@ def test_valid_config_file_with_redis() -> None:
     except Exception:
         print(traceback.format_exc())
         pytest.fail()
+
+
+def test_config_file_without_logging_config() -> None:
+    """Check how a configuration file without logging config is correctly."""
```

Comment:
"Check how a configuration file without logging config is correctly **initialized**."?

Reply:
updated